march_hardware/src/IMotionCube.cpp

```diff
@@ -16,138 +16,39 @@ IMotionCube::IMotionCube(int slaveIndex, Encoder encoder) : Slave(slaveIndex)
   this->encoder.setSlaveIndex(this->slaveIndex);
 }
 
-// TODO(Martijn Isha) change name to show this is only EtherCAT startup initialization
 void IMotionCube::writeInitialSDOs(int ecatCycleTime)
 {
   mapPDOs();
   writeInitialSettings(ecatCycleTime);
 }
 
-// Map Process Data Object by sending SDOs to the IMC
-bool IMotionCube::mapPDOs()
+// Map Process Data Object (PDO) by sending SDOs to the IMC
+void IMotionCube::mapPDOs()
```

Comment:
Split into two functions and call them both from the writeInitialSDOs

Reply:
Which two functions? adding objects and mapping+sending sdo's?

Reply:
No the MISO and MOSI mapping are different functions

Reply:
Split these into different functions in latest commit